plugins/PermissionsEx/permissions.yml

```diff
@@ -156,6 +160,7 @@ groups:
     - essentials.togglejail
     - essentials.top
     - essentials.tp
+    - essentials.tpo
```

Comment:
Was tpo not the permission to tp when a player disabled teleportation?

Instructions:	Teleports you directly to the specified player.
This command works regardless of the target players tptoggle.
Recommended for mods or above.
Permissions:	
essentials.tpo	Allow access to the /tpo command.
essentials.worlds.<worldname>	Allows players access to a specific world.